scss/settings/_settings.scss

```diff
@@ -58,7 +58,7 @@ $global-font-color: $black;
 $body-font-family: Helvetica, Arial, sans-serif;
 $global-font-weight: normal;
 $header-color: inherit;
-$global-line-height: 1.3;
+$global-line-height: 130%;
```

Comment:
Settings file is generated automatically 

Revert from here and change it here => https://github.com/zurb/foundation-emails/blob/develop/scss/components/_typography.scss#L27